src/wizard/wizard-dialog.ts

```diff
@@ -227,24 +227,33 @@ export class ESPHomeWizardDialog extends LitElement {
     let hideActions = false;
     const content = html`
       <div>
-        A device needs to be connected to a computer using a USB cable to be
-        added to ESPHome. Once added, ESPHome will interact with the device
+        A device needs to be connected to a computer (either the server running
+        ESPHome or the client running the browser) using a USB cable to be added
+        to ESPHome. Once added, ESPHome will interact with the device
         wirelessly.
       </div>
       <div>
-        ${allowsWebSerial
-          ? "Your browser does not support WebSerial."
-          : "You are not browsing the ESPHome Device Builder over a secure connection (HTTPS)."}
-        This prevents ESPHome from being able to install this on devices
-        connected to this computer.
+        The following issues prevent ESPHome from being able to remotely install
+        the necessary firmware on devices physically connected to the client
+        computer (running the browser):
+        <ul>
+          <li>Your browser does not support WebSerial.</li>
```

Comment:
This is wrong. Gate this with `supportsWebSerial` because now people with a browser that supports WebSerial will be shown an error message that is not relevant, it can only cause confusion. Does my browser support webSerial is someting we should tell them.